lib/Importer.js

```diff
@@ -154,24 +154,32 @@ class Importer {
     const unusedImportVariables = new Set();
     Object.keys(unusedVariables).forEach((variable) => {
       const lineNumbers = unusedVariables[variable];
-      if (anyNumbersWithinRange(lineNumbers, oldImports.range)) {
+      if (oldImports.range.length > 1 &&
```

Comment:
Also, I'm not really sure I understand why we need this, even after reading your commit message.